core-web/apps/dotcms-ui/src/stories/primeng/chip/templates.ts

```diff
@@ -0,0 +1,19 @@
+export const ExampleVariations = `
+<div style="display: flex; gap: 24px; flex-direction: column; align-items: center">
```

Comment:
I am moving away from this a leaving the user use the controllers... check what I did in buttons... is not set in store, we might want to discuss.

Reply:
File removed